### PR TITLE
Native Page Editor Container and UI Tests

### PR DESCRIPTION
**Pull request by mazevedofs**
**Phabricator:** https://phabricator.wikimedia.org/T347870

### Notes
* This PR brings over to the main app the contents of this PR https://github.com/wikimedia/wikipedia-ios-components/pull/9
* It also adjusts UI Tests to run in isolation from unit tests

### Test Steps
1. To test the component itself, run the app On the staging target and tap edit on an article. You should see the new editor
2. To run the UI Tests, hit CMD +U on the new WikipediaUITests target
3. Run the tests on  Xcode Cloud - manually trigger the UI Tests workflow under the main app configurations

Note: currently the theming is not working on the test plan configurations, I'm investigating that

### CODE REVIEW

File: Wikipedia.xcodeproj/xcshareddata/xcschemes/NotificationServiceExtension.xcscheme [hunk 1]
```diff
@@ -56,7 +56,8 @@
       debugDocumentVersioning = "YES"
       debugServiceExtension = "internal"
       allowLocationSimulation = "YES"
-      launchAutomaticallySubstyle = "2">
+      launchAutomaticallySubstyle = "2"
+      notificationPayloadFile = "PushNotificationPayload.apns">
```
Comment (tonisevener) [inline Wikipedia.xcodeproj/xcshareddata/xcschemes/NotificationServiceExtension.xcscheme:60]: Xcode automatically changed this on me, FYI.

**Review by tonisevener — CHANGES_REQUESTED** [review]
Nice work, it's looking good! I noticed the theming in the UI tests weren't working quite right on my machine, so I pushed a fix for that. I also reverted any UITest mentions in the other schemes that seemed unnecessary to me. Let me know if that works for you.

Unit tests are failing because of merge conflicts with `main`, but hopefully they'll run again once conflicts are fixed.

File: WMF Framework/CommonStrings.swift [hunk 2]
```diff
@@ -519,6 +519,70 @@ public class CommonStrings: NSObject {
...
+    public static let accessibilityLabelButtonCommentSelected = WMFLocalizedString("source-editor-accessibility-label-comment-selected", value: "Remove comment syntax", comment: "Accessibility label for the selected comment button on the page editor")
+    public static let accessibilityLabelButtonSuperscript = WMFLocalizedString("source-editor-accessibility-label-superscript", value: "Add superscript formatting", comment: "Accessibility label for the superscript button on the page editor")
+    public static let accessibilityLabelButtonSuperscriptSelected = WMFLocalizedString("source-editor-accessibility-label-superscript-selected", value: "Remove superscript formatting", comment: "Accessibility string for the selected superscript button on the page editor")
+    public static let accessibilityLabelButtonSubscript = WMFLocalizedString("source-editor-accessibility-label-subscript", value: "Add subscript formatting", comment: "Accessibility label for the subscript button on the page editor")
```
[the remark this answers is not in the document]
Comment (mazevedofs) [reply WMF Framework/CommonStrings.swift:567]: I prefer to check in with design on this one.

**Comment by mazevedofs** [comment]
> Nice work, it's looking good! I noticed the theming in the UI tests weren't working quite right on my machine, so I pushed a fix for that. I also reverted any UITest mentions in the other schemes that seemed unnecessary to me. Let me know if that works for you.
> 
> Unit tests are failing because of merge conflicts with `main`, but hopefully they'll run again once conflicts are fixed.

Thanks for fixing the theming, it wasn't really working, but I couldn't figure it out (and forgot to ask about it on our 1:1s)

**Review by tonisevener — APPROVED** [review]
👏 Working great!

File: Wikipedia/Code/ExploreViewController.swift [hunk 3]
```diff
@@ -1087,3 +1087,9 @@ extension ExploreViewController {
     }
 
 }
+
+ extension ExploreViewController: WKSourceEditorViewControllerDelegate {
+    func sourceEditorViewControllerDidTapFind(sourceEditorViewController: Components.WKSourceEditorViewController) {
+        // protocol conformance to run UI Tests
```
Comment (tonisevener) [inline Wikipedia/Code/ExploreViewController.swift:1093]: Sorry, I missed that this was needed. 😅

**Pull request merged**